src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -963,6 +963,10 @@ public boolean isAllowedOnStateForSpec(String stateFullName, CaseEvent caseEvent
     public boolean isAllowed(CaseDetails caseDetails, CaseEvent caseEvent) {
         CaseData caseData = caseDetailsConverter.toCaseData(caseDetails);
 
+        if (caseEvent.equals(migrateCase)) {
```

Comment:
would be nice to have a unit test for this as well

Reply:
added test